3-reveal/migrations/5-IRS/4-Zambia-2020/deploy/zambia_irs_mop_up.psql

```diff
@@ -146,11 +158,13 @@ SELECT * FROM
         zambia_jurisdictions.jurisdiction_id AS jurisdiction_id,
         zambia_jurisdictions.jurisdiction_parent_id AS jurisdiction_parent_id,
         zambia_jurisdictions.jurisdiction_name AS jurisdiction_name,
-        zambia_jurisdictions.jurisdiction_geometry AS jurisdiction_geometry,
+        --zambia_jurisdictions.jurisdiction_geometry AS jurisdiction_geometry,
```

Comment:
the views go *way* faster if we keep geometry out.  Maybe think about modifiying our views to attach the geometries as optional extensions (maybe not even materialized, not sure that would help much?) to the final views vs every view in-between?  It's a trivial join.

That's all in the future though, right now I was just trying to iterate faster and we can re-attach geometries in the final product.

Reply:
I'm completely fine with this.  We have been changing the web apps not to need the geometry from here.